### ISSUE TITLE
Accessibility: Drawer: aria-label attribute cannot be used on a span with no valid role attribute.

### ISSUE THREAD

**Issue by robertmausser** [adaptlearning/adapt-contrib-core#337]
### Subject of the issue/enhancement/features
.drawer-heading span uses aria-label attribute without valid role attribute

### Your environment
* version 5.8
* Chrome latest
* Windows 11

### Steps to reproduce
"Element Location:
.drawer-heading
<span class=""drawer-heading"" aria-label=""Top of side drawer""></span>"

### Expected behaviour
role="heading" should be applied

### Actual behaviour
No role

**Comment by oliverfoster** [comment]
Latest version:
https://github.com/adaptlearning/adapt-contrib-core/blob/7a8619cbd54f8078ccdf668f20e2894c74f14591/templates/drawer.hbs#L3

**Comment by oliverfoster** [comment]
Fixed in later version.
